__tests__/app.test.js

```diff
@@ -58,6 +58,30 @@ describe("GET /api/articles/:article_id", () => {
         );
       });
   });
+  test("200: Responds with articles sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles.length).toBe(12);
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              author: expect.any(String),
+              title: expect.any(String),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              body: expect.any(String),
+              comment_count: expect.any(String),
+            })
+          );
+        });
+      });
```

Comment:
Hey Malavika, just a couple of issues

1. You have this test within the /api/articles/:article_id describe block. Make sure to have a separate describe block to handle all /api/articles requests

2. You have amended your original 200 /api/articles test to include the later additional behaviour of an additional comment count. This is bad tdd as you are altering a previous test to account for new behaviour. You should be writing a new test for this behaviour whilst still passing the previous test. Otherwise you will lose confidence in your testing.